react/features/base/media/components/web/Audio.js

```diff
@@ -41,8 +41,11 @@ export default class Audio extends AbstractAudio {
      * @returns {ReactElement}
      */
     render() {
+        const loop = this.props.loop ? 'true' : null;
+
         return (
             <audio
+                loop = { loop }
```

Comment:
Just curious, html5 defines the loop attribute as a present-or-not attribute, so when it's present, it'll always loop the sound. I'm not familiar with how react renders html, but wouldn't it be safer to leave the loop prop entirely if the loop is false?

Reply:
If `loop === null` the attribute shouldn't be rendered and `loop` is not `null` only when there is a property loop with value `true`. So it should behave exactly as you described it!